code.js

```diff
@@ -1,3 +1,21 @@
-function binarySearch(list, element) {
+function binarySearch(list, element) {  
+    list.sort();
+    let l= 0;
+    let k = list.length-1;
+while (l <= k){
+        var mid = Math.floor((k+l)/2);
+        if (list[mid] === element){
+            return mid;
+        }
+        else if (list[mid] < element){
+            k = mid+1;
+        }
+        else{ 
+            l = mid-1;
+        }
+    }
     return -1;
 }
+
+console.log(binarySearch([1,2,3,4,5]),3);
+//For some reason I am not getting any output when trying to print out the results of the list and element form above. 
```

Comment:
You've mixed your parenthesis there and are calling the function only with the array, not the element to look for.